src/dbnode/persist/fs/commitlog/reader.go

```diff
@@ -144,49 +150,41 @@ func (r *reader) readInfo() (schema.LogInfo, error) {
 }
 
 // Read reads the next log entry in order.
-func (r *reader) Read() (
-	series ts.Series,
-	datapoint ts.Datapoint,
-	unit xtime.Unit,
-	annotation ts.Annotation,
-	err error,
-) {
-	var (
-		entry    schema.LogEntry
-		metadata seriesMetadata
-	)
-	for !metadata.passedPredicate {
```

Comment:
Do not need predicates anymore since always bootstrapping all namespaces - caller can filter if they really want to since it's now cheap to return each log entry.